### Задание 1. Вы провели эксперимент c упрощением формы заказа в магазине Утконос и получили результаты по метрике конверсий в покупку. Выберите метод оценки и оцените есть ли стат.значимые различия между конверсиями в двух группах при alpha = 5%. Дайте краткие рекомендации команде . Результаты: 
1. Число юзеров в группах , которые заходили на сайт в период эксперимента: n1 = 15550 и n2 = 15550 . 
2. Число юзеров в группах , которые совершили хотя бы одну покупку за период эксперимента: n1 = 164 и n2 = 228
3. Конверсии : conv1 = 1.05% conv2 = 1.47% . 

In [32]:

from scipy import stats
n1 = n2 = 15550
n1_buy = 164
n2_buy = 228
conv1 = 1.05/100
conv2 = 1.47/100
alpha = 0.05

#### Т.к. метрика конверсии это доля, то сравним данные доли используя z-статистику.
#### Конверсия в результате тестирования выросла, поэтому будем проводить правосторонний тест.
#### Но-гипотеза: различия в конверсиях до и после эксперимента статистически не значимы.
#### Н1-гипотеза: рост конверсии статистически значим.

In [42]:
# находим общую конверсию двух выборок
conv_total = (n1_buy + n2_buy)/ (n1 + n2)

# находим стандартную ошибку
se = ((conv_total*(1 - conv_total))*(1/n1 + 1/n2))**0.5

# находим z-критерий
z_stat = (conv2 - conv1)/se

# находим критическое значение для уровня достоверности 95% 
z_tabl = stats.norm.ppf(1-alpha)

print(z_stat)
print(z_tabl)

3.3196406865514425
1.6448536269514722


#### Сравниваем полученные значения: найденный z-критерий (3.3196406865514425) больше критического значения (1.6448536269514722), следовательно он попадает в область принятия Н1-гипотезы. Вывод: увеличение конверсии статистически значимо и необходимо внедрить упрощение формы заказа. 

### Задание 2. Сравниваем метрику конверсия в покупку. Размер выборки - 10000 элементов в каждой группе . Какой статистический критерий тут лучше всего подойдёт и почему ?



#### Размер выборки в 10000 элементов достаточно большой, чтобы использовать z-критерий. Т-критерий обычно используется при размере выборки до 100. 